### PR TITLE
If command is local, allow project admins to modify.

### PR DESCRIPTION
**Pull request by henders**
Allow a project admin to edit the local project commands.

/cc @zendesk/samson 

### References
 - Jira link: 

### Risks
 - None

### CODE REVIEW

**Comment by sbrnunes** [comment]
Sorry, another permission tweak that was missed. :+1:

**Comment by henders** [comment]
@sbrnunes : Have another look, I had missed some areas around the command pages as well.

File: app/controllers/admin/commands_controller.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 class Admin::CommandsController < ApplicationController
-  before_action :authorize_admin!
+  include ProjectLevelAuthorization
```
Comment (sbrnunes) [inline app/controllers/admin/commands_controller.rb]: Hummm, this CommandsController should get split between admin (gor global commands) and project specific one.
In the meantime, would it be better to have:

File: test/controllers/admin/commands_controller_test.rb [hunk 2]
```diff
@@ -137,4 +137,49 @@
...
+        assert_response :ok
+      end
+
+      it 'does not update for local command as non-project-admin' do
+        patch :update, id: commands(:echo).id, command: attributes, format: :json
+        @unauthorized.must_equal true, "Request was not marked unauthorized"
+      end
+    end
+  end
 end
```
Comment (sbrnunes) [inline test/controllers/admin/commands_controller_test.rb]: I would be nice to have a few extra tests checking for:
- project admins changing global commands (which only system-level admins can change, I think) 
- system level admins changing project specific commands

But I understand if you prefer to do it later to get this fixed ASAP.

**Comment by sbrnunes** [comment]
:+1 after comments have been reviewed

**Comment by grosser** [comment]
:+1: not sure why we need this since they should be able to edit inline, but still a great consistency fix!

File: test/controllers/admin/commands_controller_test.rb (hunk 2 above)
Comment (henders) [inline test/controllers/admin/commands_controller_test.rb]: There are tests there for "project admins changing global commands" which I verify is unauthorized. And there are existing tests for system level admins.

**Comment by henders** [comment]
@grosser , even editing inline needed this because the inline-edit box makes a POST to this admin controller.

File: app/controllers/admin/commands_controller.rb (hunk 1 above)
Comment (henders) [inline app/controllers/admin/commands_controller.rb]: Hmm, good call.

File: test/controllers/admin/commands_controller_test.rb (hunk 2 above)
Comment (sbrnunes) [inline test/controllers/admin/commands_controller_test.rb]: Oh, sorry, I took a very brief look but didn't see it. My bad.

File: app/controllers/admin/commands_controller.rb (hunk 1 above)
Comment (henders) [inline app/controllers/admin/commands_controller.rb]: Ah, can't seem to do that, seems the before_actions override each other in some weird way. When I tried the above, the before_action for the :index, :create, :destroy was removed and let anyone through.

**Pull request merged**